src/magic_folder/snapshot.py

```diff
@@ -723,15 +739,12 @@ def write_snapshot_to_tahoe(snapshot, author_key, tahoe_client):
     # failure, we'll try again.
     returnValue(
         RemoteSnapshot(
-            # XXX: we are copying over the name from LocalSnapshot, it is not
-            # stored on tahoe at the moment. This means, when we read back a snapshot
-            # we cannot create a RemoteSnapshot object from a cap string.
             name=snapshot.name,
             author=create_author(  # remove signing_key, doesn't make sense on remote snapshots
                 name=snapshot.author.name,
                 verify_key=snapshot.author.verify_key,
             ),
-            metadata=content_metadata,  # XXX not authenticated by signature...
+            metadata=snapshot_metadata,
             parents_raw=parents_raw,  # XXX FIXME (at this point, will have parents' immutable caps .. parents don't ork yet)
```

Comment:
So ... I think this means the parent references aren't signed?  Does that mean history is forgeable?  (eg I could take your snapshot objects and rewrite the parent pointers, and your signature would still verify?)

If so, I guess parents belong in `snapshot_metadata` instead, so this isn't possible (still could have an accessor on `RemoteSnapshot` so application code doesn't have to go digging through a dict directly).

Reply:
Hmm, yeah good point. The capability of the **Snapshot** includes the parents but yes that isn't (can't be) part of the signature of THIS snapshot.

Reply:
okay, I moved the parents to the metadata